en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,113 @@
+The WSO2 API Manager consists of 6 main components,
+
+* [API Publisher](#api-publisher)
+* [API Developer Portal](#api-developer-portal)
+* [API Gateway](#api-gateway) (now comes as a [API Microgateway](https://docs.wso2.com/display/MG300/API+Microgateway+Documentation) as well)
+* [Key Manager](#key-manager)
+* [Traffic Manager](#traffic-manager)
+* [API Analytics](#analytics)
+
+A component is made up of one or more [OSGi](http://www.osgi.org/Technology/Home) bundles. A bundle is the modularization unit in OSGi, similar to a JAR file in Java.
 
 The component architecture of the WSO2 API Manager is illustrated in the diagram below.
 
 #### API Publisher
 
-![]({{base_path}}/assets/attachments/103327648/126556771.png)
-
-API development is usually done by someone who understands the technical aspects of the API, interfaces, documentation, versions etc., while API management is typically carried out by someone who understands the business aspects of the APIs. In most business environments, API development is a responsibility that is distinct from API publication and management.
+![API Publisher Component Architecture]({{base_path}}/assets/attachments/103327648/126556771.png)
 
-WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizating APIs, and promoting.
+API development is usually done by someone who understands the technical aspects of the API, interfaces, documentation, versions, etc., while API management is typically carried out by someone who understands the business aspects of the APIs. In most business environments, API development is a responsibility that is distinct from API publication and management.
 
-The diagram above shows the common lifecycle activities of an API developer/manager.
+WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizing APIs, and promoting.
 
-#### Developer Portal
+The diagram above shows the common lifecycle activities of an API developer/manager.
 
-![]({{base_path}}/assets/attachments/103327648/126556772.png)
+#### API Developer Portal
 
```

Comment:
The change doesn't make a difference.

Reply:
As in the paragraph, we have introduced the `Devportal` as `API Developer Portal`. Therefore, keeping the same naming style across the docs for more clarity

Reply:
noted.